diff -rbBU3 FreeRDP-2.0.0-rc4-orig/cmake/FindFFmpeg.cmake FreeRDP-2.0.0-rc4/cmake/FindFFmpeg.cmake
--- FreeRDP-2.0.0-rc4-orig/cmake/FindFFmpeg.cmake	2020-02-28 00:28:17.260613596 +0900
+++ FreeRDP-2.0.0-rc4/cmake/FindFFmpeg.cmake	2020-02-28 00:28:22.860561160 +0900
@@ -19,7 +19,7 @@
 
 # avcodec
 find_path(AVCODEC_INCLUDE_DIR libavcodec/avcodec.h PATHS ${AVCODEC_INCLUDE_DIRS})
-find_library(AVCODEC_LIBRARY avcodec PATHS $ {AVCODEC_LIBRARY_DIRS})
+find_library(AVCODEC_LIBRARY avcodec PATHS ${AVCODEC_LIBRARY_DIRS})
 
 # avutil
 find_path(AVUTIL_INCLUDE_DIR libavutil/avutil.h PATHS ${AVUTIL_INCLUDE_DIRS})
