diff -rbBU3 cmake-3.16.4-orig/Source/cmFindPackageCommand.cxx cmake-3.16.4/Source/cmFindPackageCommand.cxx
--- cmake-3.16.4-orig/Source/cmFindPackageCommand.cxx	2020-02-27 23:32:23.671885416 +0900
+++ cmake-3.16.4/Source/cmFindPackageCommand.cxx	2020-02-27 23:40:34.377286781 +0900
@@ -482,6 +482,7 @@
 
     // Add root paths from <PackageName>_ROOT CMake and environment variables,
     // subject to CMP0074.
+#if 0
     switch (this->Makefile->GetPolicyStatus(cmPolicies::CMP0074)) {
       case cmPolicies::WARN:
         this->Makefile->MaybeWarnCMP0074(this->Name);
@@ -496,14 +497,17 @@
           cmPolicies::GetRequiredPolicyError(cmPolicies::CMP0074));
         break;
       case cmPolicies::NEW: {
+#endif
         // NEW behavior is to honor the <pkg>_ROOT variables.
         std::string const rootVar = this->Name + "_ROOT";
         if (const char* pkgRoot = this->Makefile->GetDefinition(rootVar)) {
           cmExpandList(pkgRoot, rootPaths, false);
         }
         cmSystemTools::GetPath(rootPaths, rootVar.c_str());
+#if 0
       } break;
     }
+#endif
   }
 
   this->SetModuleVariables(components);
